### DISCUSSION TITLE
是否是腾讯的反爬虫？

### DISCUSSION THREAD

**Discussion by To-phoenix-zhw** [Algebra-FUN/WeReadScan#26] [General]
[竞争战略.pdf](https://github.com/Algebra-FUN/WeReadScan/files/9036206/default.pdf)
降低selenium版本或者重写S函数之后，成功把demo运行起来了，并下载得到了《竞争战略》的pdf。
然而里面没有文字。如果是类似于截屏的扫描，应该有文字显示出来才对。
这个问题是否是腾讯的反爬虫？我们应该如何解决呢？
请项目主抽空看一看。
![image](https://user-images.githubusercontent.com/48126062/177076504-a9f8ac08-5219-45d0-899c-88753ff9c12c.png)

**Comment by Algebra-FUN** [comment]
确实是有通过使用特定字库进行反爬虫的操作，但是今天我自己运行了一遍，并没有出现这样的问题，所以应该不是反爬虫操作造成的。

这个问题可能是由于你的设备本身缺少字库造成的。
可以直接运行

```python
from selenium.webdriver import Chrome

driver = Chrome()
driver.get('https://weread.qq.com/web/reader/50532a905cde3050538da2b')
input()
```

观察是否将文字渲染了出来
